service/kvmconfig/v6/key/key.go

```diff
@@ -140,6 +143,41 @@ func MemoryQuantity(n v1alpha1.KVMConfigSpecKVMNode) (resource.Quantity, error)
 	return q, nil
 }
 
+// MemoryQuantity returns a resource.Quantity that represents the memory to be used by the nodes.
+// It adds the memory from the node definition parameter to the additional memory calculated on the node role
+func MemoryQuantityWorker(n v1alpha1.KVMConfigSpecKVMNode) (resource.Quantity, error) {
+	mQuantity, err := resource.ParseQuantity(n.Memory)
+	if err != nil {
+		return resource.Quantity{}, microerror.Maskf(err, "calculating memory overhead multiplier")
+	}
+
+	// base worker memory calculated in MB
+	q, err := resource.ParseQuantity(fmt.Sprintf("%dM", mQuantity.ScaledValue(resource.Giga)*1024))
+	if err != nil {
+		return resource.Quantity{}, microerror.Maskf(err, "creating Memory quantity from node definition")
+	}
+	// IO overhead for qemu is around 512M memory
+	IOOverhead := resource.MustParse(workerIOOverhead)
+	q.Add(IOOverhead)
```

Comment:
Nitpick: golang vairiables start with lowercase letter, so `ioOverhead`.

Reply:
yeah i was thinking about that, IO is usually written with uppercase so wasnt sure, but its not really issue